.github/workflows/Jirabot.yml

```diff
@@ -45,8 +45,17 @@ jobs:
             import time
             import sys
             import json
+            import subprocess
+            from email.utils import parseaddr
             from atlassian.jira import Jira
 
+            def sanitizeInput(input: str, inputType: str) -> str:
+                if inputType.lower() == 'email':
+                    # Return the email address only, returns '' if not valid or found
+                    return parseaddr(input)[1]
+                else:
+                    return input
```

Comment:
this could be considered a false positive, considering we don't know what format the input represents, it should return '' or an outright failure since email type is the only supported format.

Reply:
Good point